app/screens/home/index.tsx

```diff
@@ -78,17 +78,17 @@ export default function HomeScreen(props: HomeProps) {
                 <Tab.Screen
                     name={Screens.SEARCH}
                     component={Search}
-                    options={{unmountOnBlur: false, tabBarTestID: 'tab_bar.search.tab'}}
+                    options={{unmountOnBlur: false, lazy: true, tabBarTestID: 'tab_bar.search.tab'}}
                 />
                 <Tab.Screen
                     name={Screens.MENTIONS}
                     component={RecentMentions}
-                    options={{tabBarTestID: 'tab_bar.mentions.tab'}}
+                    options={{tabBarTestID: 'tab_bar.mentions.tab', lazy: true, unmountOnBlur: false}}
                 />
                 <Tab.Screen
                     name={Screens.ACCOUNT}
                     component={Account}
-                    options={{tabBarTestID: 'tab_bar.account.tab'}}
+                    options={{tabBarTestID: 'tab_bar.account.tab', lazy: true, unmountOnBlur: false}}
```

Comment:
We are setting `lazy` to true because we just want it to load when we get to it and then `unmountOnBlur` to false, because now, `react-freeze` takes care of not re-rendering components that are not visible to the user. right?

Reply:
Correct